### PR TITLE
#25 Add startApp.js to use the express app return by app.js with spec…

### PR DESCRIPTION
**Pull request by acabunoc**
…ific config return by config.js

### CODE REVIEW

**Comment by acabunoc** [comment]
PR for changes by @aid29! (thanks again, Ai)

File: conf.default.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (acabunoc) [inline conf.default.json]: I think we have this info in env.dist -- did you meant to commit this?

https://github.com/mozillascience/PaperBadger/blob/master/env.dist

Comment (aid29) [inline conf.default.json]: I think json file is here to help give the default config to app without setup the environment variables. 
If we don't need env.dist to deploy the app on Heroku, I think we can delete it. I will update the README.md for the changes.

Comment (acabunoc) [inline conf.default.json]: Yeah, I just saw you stuff in src/config.js (sry had some mtgs mid-review)

please update the README, I'm on the fence on storing config as json. Is there a reason you didn't want to use something like dotenv? https://www.npmjs.com/package/dotenv

Comment (aid29) [inline conf.default.json]: Understood, i'm ok to use dotenv. i will make the commit for dotenv.

Comment (acabunoc) [inline conf.default.json]: The less code we have to maintain the better ;)

**Comment by acabunoc** [comment]
This looks great, thanks @aid29!
r+

**Pull request merged**